flowzone.yml

```diff
@@ -2183,6 +2185,18 @@ jobs:
             echo  "::error::Failed to get branch protection rules ${message} ${result} "
             exit 1
           fi
+
+      # disable auto-merge on any existing PRs if the required reviewer count is decreasing
```

Comment:
How does this know the value is decreasing ? 

Does it mean to say "check if the required reviews from Github is less than Flowzone specified required reviews" ?